src/NUnitEngine/nunit.engine/Services/DomainManager.cs

```diff
@@ -120,7 +118,7 @@ AppDomainSetup CreateAppDomainSetup(TestPackage package)
                 }
             }
 
-            if (package.GetSetting("ShadowCopyFiles", false))
+            if (package.GetSetting(PackageSettings.ShadowCopyFiles, false))
             {
```

Comment:
ShadowCopyFiles came up as only being set in console which seemed odd - this should be using the setting rather than a string.